encoding/hcl.go

```diff
@@ -7,14 +7,46 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"strings"
 
 	"github.com/rodaine/hclencoder"
 )
 
-func HCL(in interface{}) io.Reader {
+// hclEncoder implements the Encoder interface.
+type hclEncoder struct {
+	io.Reader
+}
+
+// EncodeComment returns byte slice that represents a HCL comment.
+// We split `lines` by U+000A and encode as a single/multi line comment.
+func (hclEncoder) EncodeComment(lines string) []byte {
+	commentLines := strings.Split(lines, "\n")
```

Comment:
Wanna use `return YAML("").Encode(lines)`? It's the same (:  (reuse all)

Reply:
Yeah I was wondering if that was a problem as HCL also has a separate multiline syntax too. But would be better to reuse. 🙂